CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Breaking Changes
 - Removal of the `dist_threshold` flag and in the `-toml` file.
+- The `uhf` is now set to 0 if `xtb` is used and a lanthanide is within the molecule, to match the `f in core` approximation.
```

Comment:
```suggestion
- The number of unpaired electrons (`Molece.uhf`) is now set to 0 if `xtb` is used as `QMMethod` and a lanthanide is within the molecule to match the `f-in-core` approximation.
```

Reply:
Before i commit this change is the (`Molece.uhf) a typo?

Reply:
Yes! `Molecule.uhf`